src/language-js/printer-estree.js

```diff
@@ -802,10 +802,16 @@ function printPathNoParens(path, options, print, args) {
 
       const optional = printOptionalToken(path);
       if (
-        // We want to keep require calls as a unit
+        // We want to keep CommonJS require calls as a unit
         (!isNew &&
           n.callee.type === "Identifier" &&
           n.callee.name === "require") ||
+        // As well as AMD-style require and define calls.
+        // Note that the previuos case covers AMD-style require calls as well.
+        // e.g. `define(["some/lib", (lib) => {`
+        (!isNew &&
+          n.callee.type === "Identifier" &&
+          n.callee.name === "define") ||
```

Comment:
What do you think about moving this to the previous test?

```js
(!isNew &&
  n.callee.type === "Identifier" &&
  (n.callee.name === "require" ||
    n.callee.name === "define"))
```

Reply:
I did that at first; I'm ambivalent there honestly. I ended up with this approach because it allows for a more obvious way to diverge CommonJs calls from AMD calls in the future. Lemme know and I'm down to move it to the previous test.

Reply:
Let's move it into the previous test